### PR TITLE
add docker-cloud.yml to the top of the list of stack filenames

### PR DESCRIPTION
**Pull request by tifayuki**

### CODE REVIEW

File: dockercloudcli/utils.py [hunk 1]
```diff
@@ -304,18 +304,15 @@ def load_stack_file(name, stackfile, stack=None):
...
-            filematch += 1
         if os.path.exists("docker-compose.yml"):
             stackfile = "docker-compose.yml"
-            filematch += 1
-        if filematch == 0:
+        if os.path.exists("tutum.yml"):
+            stackfile = "tutum.yml"
+        if os.path.exists("docker-cloud.yml"):
+            stackfile = "docker-cloud.yml"
+        if not stackfile:
```
Comment (bernardopericacho) [inline dockercloudcli/utils.py]: Wouldn't  be better to do a for, in order to maintain compatibility and the ability to add more filenames in the future?
```
for stackfile in ["docker-cloud.yml", "tutum.yml", "docker-compose.yml", "fig.ml"]:
  if os.path.exists(stockpile):
    break
if not stackfile ....
```

Comment (amegianeg) [inline dockercloudcli/utils.py]: +1 to Bernardo's: also, if the highest priority file exists (docker-cloud.yml), you do not need to look for the other ones first

File: dockercloudcli/utils.py [hunk 2]
```diff
@@ -303,27 +303,19 @@ def load_stack_file(name, stackfile, stack=None):
...
+        try:
+            interpolated_content = interpolate_environment_variables(content, 'service')
+        except:
+            raise BadParameter("Bad format of the stack file: %s" % stackfile)
 
         services = []
         if interpolated_content:
             for k, v in interpolated_content.items():
-                print ()
+                print()
```
Comment (bernardopericacho) [inline dockercloudcli/utils.py]: remove this print, it is not needed

**Comment by bernardopericacho** [comment]
Squash commits into 1

**Comment by bernardopericacho** [comment]
LGTM with comments

**Pull request merged**